app/assets/stylesheets/shared/select.scss

```diff
@@ -0,0 +1,73 @@
+.sn-select {
+  cursor: pointer;
+  position: relative;
+  padding: .5em;
+  border: 1px solid;
+  border-radius: 4px;
+  border-color: var(--sn-grey);
+
+  .sn-select__value {
+    all: unset;
```

Comment:
You need to cover long values, so it must be `...` 
And do we need different color for placeholder?